audio-worklet/index.html

```diff
@@ -38,11 +38,12 @@ <h1>
         <p>
           <a href="https://webaudio.github.io/web-audio-api/#AudioWorklet"
           target="_blank">AudioWorklet</a> is currently available on
-          <strong>Chrome Beta</strong> behind the experimental flag or via 
-          <a href=
+          <strong>Chrome Stable (Version 64)</strong> via <a href=
```

Comment:
I think I'd just say "Chrome 64 (or later)" so you don't ever have to change this again.